bats/tests/helpers/defaults.bash

```diff
@@ -156,6 +156,19 @@ validate_enum RD_9P_PROTOCOL_VERSION 9p2000 9p2000.u 9p2000.L
 
 validate_enum RD_9P_SECURITY_MODEL passthrough mapped-xattr mapped-file none
 
+########################################################################
+# When RD_USE_RAMDISK is true, we will try to set up a temporary ramdisk
+# for the application profile to make things run faster.  This is not
```

Comment:
`profile` feels like the wrong word, but "data directory" is not quite right either. I guess we can leave it, if you can't think of a better word either.